docs/guides/parallel-scraping/parallel-scraper.mjs

```diff
@@ -0,0 +1,107 @@
+import { fork } from 'node:child_process';
```

Comment:
Just wondering if [node:cluster](https://nodejs.org/api/cluster.html) has been evaluated as solution ? Seems to be a native solution to manage master process forking multiple worker processes.

There's also [node:worker_threads](https://nodejs.org/api/worker_threads.html) that seems to enable the use of multiple threads within a single process. Might be worth mentioning as an alternative in the docs maybe ?

Reply:
This is strictly an example way to do it! It's definitely not a "do it this way only"! Also, cluster could be used instead, same with worker threads, but that's more up to users to figure out